### DISCUSSION TITLE
About Multilevel Navigation

### DISCUSSION THREAD

**Discussion by xiwanzi** [requarks/wiki#7167] [Help / Questions] [answered]
In the future will there be multi-layer navigation? Or is there already a plugin out there for such community? Multi-layer navigation is like a file folder, it could finely categorize different content, so the user could simply rely on the navigation to find the type of wiki they're looking for, and then find the corresponding documentations, wouldn't that be convenient? And now the Tree Navigation and Static Navigation can only visit fixed pages, and there's no way to sort. My wiki project is a game wiki open to all, so this seems pretty important to me...

**Comment by NGPixel** [comment, marked as answer]
This has already been added in the upcoming v3.

See https://beta.js.wiki/blog/2023-wiki-js-3-feature-preview-navigation-search

**Reply by xiwanzi** [reply]
I really appreciate your reply, according to the page description, there will be Beta version in 2024Q2? Hope it will come early and make me upgrade with peace of mind! Thanks to the author and the open source community again
